input/vulnerabilities/levitator.json

```diff
@@ -30,5 +30,11 @@
       "component" : "platform/libcore/",
       "tag" : "android-2.3.6_r1"
     }
-  }
+  },
+  "Surface": ["local", "app"],
+  "Vector": ["memory-corruption"],
+  "Target": ["kernel"],
+  "Channel": ["app-execution"],
+  "Condition": ["affected-app-installed", "unknown-source-install-allowed"],
```

Comment:
is "unknown-source-install-allowed" a precondition?

Reply:
Not really - now changed